language-design.md

```diff
@@ -59,7 +135,17 @@ What does the post on grayscale tell us about the process of API design?
 
 **Response**
 
-
+The post on grayscale tells us that the community contributes to the process
+of API design. In the post, users were able to vote on which naming to use
+for grayscale. It also shows that people in the community have varied (and
+sometimes strong) opinions on things as simple as naming a function for 
+grayscale. We also were rather surprised that the rgb function won by so many
+votes since we both thought that black was by far the clearest naming choice.
+This reveals that different communities have different shared ideas that make
+some choices clearer to members of the community but not to outsiders.  We
+suspect that in the community using the rgb function is pretty common so to them
+it would seem natural to use this for a function that does not really change red
+green or blue colors but rather shades of black and white.
```

Comment:
It is very interesting that the API design was presented publicly. We disagree that the community contributes as the community voted (albeit it was a plurality not pure majority) for the `rgb()` function, the committee on CSS still chose something else that had significantly fewer votes. At the same time, it is hard to say such a loosely monitored vote (you didn't have to put in a name or be of some kind of pedigree/background to vote, etc.) should definitely have a large effect on the future of any widely used language or technology.